### PR TITLE
Remove obsolete travis badge

### PR DESCRIPTION
**Pull request by jugmac00**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,13 +1,6 @@
 Juju Charm Tools
 ================
 
-|travis| |build|
-
-.. |travis| image:: https://travis-ci.org/juju/charm-tools.svg
-    :target: https://travis-ci.org/juju/charm-tools
-.. |build| image:: https://build.snapcraft.io/badge/juju/charm-tools.svg
-    :target: https://build.snapcraft.io/user/juju/charm-tools
-
```
Comment (ajkavanagh) [inline README.rst:10]: i think the `build` target could be replaced with:

```
[![charm](https://snapcraft.io/charm/badge.svg)](https://snapcraft.io/charm)
```

e.g. [charm](https://snapcraft.io/charm)

**Review by ajkavanagh — CHANGES_REQUESTED** [review]
Good idea to remove the travis badge; however, let's keep the snap build; I've suggested the substitute url.

File: README.rst (hunk 1 above)
Comment (jugmac00) [reply README.rst]: Thank you! I was not aware of the new badge/location.

**Review by ajkavanagh — APPROVED** [review]
Thanks very much for the updates.

**Pull request merged**